.travis.yml

```diff
@@ -1,12 +1,14 @@
+sudo: false
 language: python
+cache: pip
```

Comment:
So I say either take it out or implement what the fellas in ecommerce did with the specific directory. Since we use lxml, I'd suggest the latter.